src/Symfony/Component/Messenger/Transport/RedisExt/Connection.php

```diff
@@ -43,6 +44,10 @@ class Connection
 
     public function __construct(array $configuration, array $connectionCredentials = [], array $redisOptions = [], \Redis $redis = null)
```

Comment:
unrelated, but we missed making this class compatible with other kind of redis instances: RedisArray, RedisCluser, RedisProxy and RedisClusterProxy

Reply:
are they compatible with redis streams ?

Reply:
I think so, because in the end this is just a key. https://redis.io/topics/streams-intro also mentions clusters. But I didn't try, so maybe not. Yet the hardcoded signature would be a blocker for the future.

Reply:
I will give it a try and drop the typehint in favor of a check anyways.